cpp/cert/src/rules/CTR55-CPP/DoNotUseAnAdditiveOperatorOnAnIterator.ql

```diff
@@ -15,23 +15,44 @@ import cpp
 import codingstandards.cpp.cert
 import codingstandards.cpp.Iterators
 
-from ContainerIteratorAccess it
+/**
+ * any `.size()` check above our access
+ */
+predicate size_checked_above(ContainerIteratorAccess it, IteratorSource source) {
```

Comment:
```suggestion
predicate isSizeCheckedAbove(ContainerIteratorAccess it, IteratorSource source) {
```

Reply:
How does this compare to `size_compare_bounds_checked`, or what is the intended difference?